MapboxCoreNavigation/RouteController.swift

```diff
@@ -202,6 +202,11 @@ open class RouteController: NSObject {
         DispatchQueue.main.async {
             self.startEvents(route: route)
         }
+        
+        guard let _ = Bundle.main.bundleIdentifier else { return }
```

Comment:
Framework tests do not have a "working" Info.plist- values added to it are not actually respected since its not an app.